website/docs/r/iam_account_settings.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
1)Can you please add subcategory -> Account Management (any name you find it relevant just sub category the docs together in registry docs https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs) or it should come under existing sub-category (Identity & Access Management (IAM)) 
2)remove sidebar_current
3)Add that new category into allowed-subcategories.txt  file if you are using any other name apart from exisitng subcategories

Reply:
I've made these updates here as well as in the new data source md file